test/ActiveRecordFindTest.php

```diff
@@ -14,62 +15,47 @@ class ActiveRecordFindTest extends DatabaseTestCase
 {
     public function testFindWithNoParams()
     {
-        $this->expectException(RecordNotFound::class);
+        $this->expectException(ValidationsArgumentError::class);
         Author::find();
     }
 
     public function testFindReturnsSingleModel()
```

Comment:
We probably still have to keep all these old tests for backwards compatibility purposes. The new tests are necessary too for the new syntax.

Reply:
No BC. 👿 

As far as I'm concerned, this is a complete rewrite. If people want to use the old version, they don't need to update their dependency. If they want to use the new version, they must update their usage.